cdi/src/main/java/org/hibernate/validator/cdi/internal/ValidatorFactoryBean.java

```diff
@@ -252,7 +255,18 @@ private ConstraintValidatorFactory createConstraintValidatorFactory(Configuratio
 
 		if ( constraintValidatorFactoryFqcn == null ) {
 			// use default
-			return createInstance( InjectingConstraintValidatorFactory.class );
+			InjectingConstraintValidatorFactory validatorFactory = createInstance( InjectingConstraintValidatorFactory.class );
+
+			Set<Class<? extends ConstraintValidator<?, ?>>> builtInConstraintValidators;
+			if ( config instanceof PredefinedScopeConfigurationImpl ) {
```

Comment:
BTW, the predefined validator factory and related classes are for Quarkus, and it does not use this CDI extension so IDK if we should even consider this case here ...